app/controllers/foods_controller.rb

```diff
@@ -16,7 +16,7 @@ def new
 
   def create
     @food = Food.new(food_params)
-    @user = current_user
+    @food.user = current_user
     if @food.save
       redirect_to @food, notice: 'Food was successfully created.'
     else
```

Comment:
@Alfredbis29 very nice,

Reply:
@vic778  thanks so much for the review , I think now I can move one to another milestone of this project